Celeste.Mod.mm/Mod/Everest/EverestModule.cs

```diff
@@ -206,6 +206,19 @@ public abstract class EverestModule {
                         .Change(v => prop.SetValue(settings, v))
                         .NeedsRelaunch(needsRelaunch)
                     ;
+                } else if (propType.IsEnum) {
+                    Array enumValues = Enum.GetValues(propType);
+                    Array.Sort((int[]) enumValues);
+                    string enumNamePrefix = $"{nameDefaultPrefix}{prop.Name.ToLowerInvariant()}_";
+                    item =
+                        new TextMenu.Slider(name, (i) => {
+                            string enumName = enumValues.GetValue(i).ToString();
+                            string fullName = $"{enumNamePrefix}{enumName.ToLowerInvariant()}";
+                            return fullName.DialogCleanOrNull() ?? enumName;
```

Comment:
`enumName.SpacedPascalCase()` would result in `SomeTextHere` becoming `Some Text Here` in the menu.